jerry-core/ecma/builtin-objects/ecma-builtin-helpers-date.cpp

```diff
@@ -1004,13 +992,14 @@ ecma_date_object_to_string (ecma_value_t this_arg, /**< this argument */
     ecma_object_t *obj_p = ecma_get_object_from_value (this_arg);
     ecma_property_t *prim_value_prop_p = ecma_get_internal_property (obj_p,
                                                                      ECMA_INTERNAL_PROPERTY_PRIMITIVE_NUMBER_VALUE);
```

Comment:
I would add a JERRY_ASSERT (prim_value_prop_p != NULL); even if it is unlikely to trigger.